beets/importer.py

```diff
@@ -1246,6 +1250,11 @@ def lookup_candidates(session, task):
 
     plugins.send('import_task_start', session=session, task=task)
     log.debug(u'Looking up: {0}', displayable_path(task.paths))
+
+    # Restrict the initial lookup to IDs specified by the user via the -m
+    # option. Currently all the IDs are passed onto the tasks directly.
+    task.musicbrainz_ids = session.config['musicbrainz_ids'].as_str_seq()
```

Comment:
Looks great for now. As you note in your other comment, we can move this around later if we want a more sophisticated way of populating this.